### DISCUSSION TITLE
[HELP] Customization regarding the Taskbar

### DISCUSSION THREAD

**Discussion by mustafatoktas** [valinet/ExplorerPatcher#3413] [Q&A]
I want the start button on the taskbar to be in the left corner, but the applications to be aligned in the middle.

With which application can I do this? I would be glad if you help.

Windows 11 23H2 22631.3672 

**except start 11 app. Because it's too heavy**
**I used TaskbarX, but I don't think it has such a setting.**

**Comment by pyrates999** [comment]
what do you mean start11 app is too heavy?

**Reply by mustafatoktas** [reply]
This is not an answer

**Reply by pyrates999** [reply]
Seems you already figured out to get what you want, you would install the start 11 app.  I'm just wondering why you don't want to use it.  Hence, why I asked what you mean by it being too heavy.

**Reply by mustafatoktas** [reply]
@pyrates999 
While I was using Windows 10 before, I purchased and installed start11 v2 from Steam. My goal was to make the Windows 10 taskbar like Windows 11, but when the PC was turned on, the taskbar appeared too late.

Then I had already upgraded to windows 11.

Explorer Patcher is not like start11 v2. As soon as the PC turns on, all settings instantly appear as I set them.

That's why I don't like the performance of start11 v2 and I don't want to use it.

**Note:** My system is also fine. My system specs are written in my Github profile.

**Reply by pyrates999** [reply]
Makes sense.

**Comment by mustafatoktas** [comment]
Start 11 v2 allows me to do exactly what I want, but its performance is very bad.

**I want this:** The start button is on the left but the applications are in the middle

![image](https://github.com/valinet/ExplorerPatcher/assets/52534609/d641d98e-74d5-4c4a-856a-12504ff78a9a)

**Comment by mustafatoktas** [comment]
By the way, is Start 11 v2 compatible with Explorer patcher?
Can I use both at the same time?
@Amrsatrio

**Comment by Amrsatrio** [comment]
@mustafatoktas 
<img width="1920" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/20662640/c00ea0ec-3cc3-4752-9b20-0a5917591ace">

I cannot comment on compatibility with paid and closed source programs, sorry.

**Comment by mustafatoktas** [comment]
@Amrsatrio Unfortunately, explorer patcher does not have the same options for Windows 11 taskbar settings.

![image](https://github.com/valinet/ExplorerPatcher/assets/52534609/084cf6ed-025a-419c-971b-53ffe070bc60)

**Reply by Amrsatrio** [reply]
Yeah unfortunately, ExplorerPatcher is not for that then...
